packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -215,7 +215,7 @@ export function getWebpackConfig(
         },
         {
           test: /\.(js|tsx?)$/,
-          exclude: /node_modules/,
+          exclude: /node_modules\/(?!(mermaid\/dist)\/).*/,
```

Comment:
`mermaid` is shipping as `ESM` and it seems their dist still need to be compiled 😞